ranger/data/scope.sh

```diff
@@ -92,13 +92,24 @@ handle_extension() {
 }
 
 handle_image() {
+    # Size of the preview if there are multiple options or it has to be rendered
+    # from vector graphics. If the conversion program allows specifying only one
+    # dimension while keeping the aspect ratio, the width will be used.
+    local DEFAULT_SIZE="1920x1080"
+
     local mimetype="${1}"
     case "${mimetype}" in
         # SVG
         # image/svg+xml)
-        #     convert "${FILE_PATH}" "${IMAGE_CACHE_PATH}" && exit 6
+        #     convert -- "${FILE_PATH}" "${IMAGE_CACHE_PATH}" && exit 6
         #     exit 1;;
 
+        # DjVu.
+        # image/vnd.djvu)
+        #     ddjvu -format=tiff -quality=90 -page=1 -size="${DEFAULT_SIZE}" \
+        #           - "${IMAGE_CACHE_PATH}" <"${FILE_PATH}" \
```

Comment:
Yes, in this case the hyphen means "read the DjVu file from standard input". I used it together with a redirection in case the file name starts with a dash (`ddjvu` doesn't support the `--` syntax for terminating options).

Reply:
Ah, of course it's stdin. That's what happens when you're reviewing code while being underslept.